scripts/.eslintrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require('@substrate/dev/config/eslint');
```

Comment:
Is eslint smart enough to look in the directory above?

Reply:
Yes! 100% is. TIL :)

removed all the eslint configs from the scripts folder.

Reply:
except the eslintignore